src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,27 +23,21 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> messages = new ArrayList<String>();
-
   
-  @Override
-  public void init() {
-    messages.add("Hello, how are you?");
-    messages.add("Hello, what is your name?");
-    messages.add("Hello, what is your age?");
-  }
+  SmallCityService smallCityService;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String name = "Abakir";
-    
-    response.getWriter().println(convertToJsonUsingGson(messages));
+    // TODO: Return businessList from smallCityService as JSON
   }
-  private String convertToJsonUsingGson(ArrayList<String> temp) {
-    Gson gson = new Gson();
-    String json = gson.toJson(temp);
-    return json;
+
+    @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // TODO: Create smallCityService instance from mapLocation inputted location communicated through JSON
+    // TODO: Convert lat and long maps variable to a mapLocation object
+    smallCityService = new SmallCityService(geolocation);
```

Comment:
I don't think this builds. It looks like geolocation is never defined.

Reply:
It's not defined yet - that was more of a placeholder because we need to write more of the javascript to know where to fetch the lat and long coordinates from. Should we take that construction out all together until we write more code?

Reply:
Gotcha. Since that's the case, let's take it out, or convert it to a comment. Because this is a shared repo, and serves as the source of truth for the project, only code that works should be submitted.

Reply:
I just deleted it. Going to make issues for the todos now.